### DISCUSSION TITLE
Spell Check problem for Windows 11

### DISCUSSION THREAD

**Discussion by lesdelc** [jendrikseipp/rednotebook#728] [Q&A]
I am using RedNotebook version 2.30 for Windows 11 and did download Aspell (Aspell-en-0.50-2-3.exe) and installed it. After selecting Spell Check, Aspell underlines ALL text in RED instead of the misspelled word. How can I resolve this problem?

**Comment by jendrikseipp** [comment]
RedNotebook should be able to do spell checking without any external tools such as aspell. If it doesn't, that's a bug.

**Reply by lesdelc** [reply]
Thanks for the prompt reply. I never expected Spell Check to work without external tools.
I again visited the Aspell website and noticed a new version of Aspell (Aspell-en-0.50-3-3-Setup.exe). Installed the new version, and Aspell is working great. I am enjoying the app. By the way, I am a long-time user of Linux; however, I do need Windows for some programs.

**Comment by jendrikseipp** [comment]
Glad to hear it works now.

**Reply by lesdelc** [reply]
Your comment about external tools is correct. In the editor when I right-click a menu appears. I selected Languages the default is de_DE (German). I corrected this and selected en_US. This cleared ALL text in RED and pointed out my spelling arrears.
